docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md

```diff
@@ -74,7 +74,7 @@ As a result, you need to include the SDK artifacts as follows in Maven:
 
 The SDK provides a [pre-packaged runtime environment](/components/connectors/custom-built-connectors/connector-sdk.md#pre-packaged-runtime-environment)
 that you can start manually. With version 0.3.0, this runtime moves from the [SDK repository](https://github.com/camunda/connector-sdk/tree/stable/0.2/runtime-job-worker)
-to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe/tree/master/connector-runtime). This also means that the provided runtime now is
+to [Spring Zeebe](https://github.com/camunda/connectors/blob/main/connector-runtime/README.md). This also means that the provided runtime now is
```

Comment:
Oh goodness. I know the Connector runtime was still a part of the project in [8.2](https://github.com/camunda-community-hub/spring-zeebe/tree/8.2.0/connector-runtime) but it disappears in 8.3. I _think_ linking to 8.2 is more appropriate here, but you may want to open a backlog item to discuss cleaning up these update guides with @sbuettner and team.

I don't have a clear mapping of what Connector version goes to what C8 version to suggest it myself without digging.

Reply:
@akeller As Connectors was only officially released with 8.3 and some of the links are outdated anyway (connectors-sdk was for exampled also moved into the mono repo) should we maybe clean up the docs and remove the links?

Reply:
@sbuettner great idea. Can you sync with @christinaausley on this and open a separate PR?

For this PR, would my suggestion be ok? Or keep the change as-is?

Reply:
Shouldnt the name be rather `Connector Runtime`?

Reply:
Adjusted in latest commit 👍 Let me know how I can assist with the clean-up.